### PR TITLE
Next: Refactoring to support Nette v3

### PR DESCRIPTION
**Pull request by mabar**

### CODE REVIEW

File: src/DI/LatteConsoleExtension.php [hunk 1]
```diff
@@ -6,45 +6,52 @@
...
-	private $defaults = [
-		'warmup' => [
-			'%appDir%',
-		],
-		'warmupExclude' => [],
-		'purge' => [
-			'%tempDir%/cache/latte',
-		],
-	];
+	public static function createSchema(): Schema
```
Comment (f3l1x) [inline src/DI/LatteConsoleExtension.php:19]: Why static method here?

Comment (mabar) [reply src/DI/LatteConsoleExtension.php:19]: We need schema also in [ConsoleBridgesExtension](https://github.com/contributte/console-extra/blob/next/src/DI/ConsoleBridgesExtension.php#L19)
I can split registration of extensions into two phases (__construct() and loadConfiguration()) to make it non-statical

Comment (mabar) [reply src/DI/LatteConsoleExtension.php:19]: Better to keep it statical. In __construct() we don't know if extension should be instantiated or not.

File: composer.json [hunk 2]
```diff
@@ -20,19 +20,20 @@
     }
   ],
   "require": {
-    "php": ">= 7.1",
-    "nette/di": "~2.4.11|~3.0",
+    "php": "^7.2",
+    "contributte/di": "dev-next",
```
Comment (mabar) [inline composer.json]: Must be changed before merge
https://github.com/contributte/di/pull/15

**Pull request merged**